CHANGELOG.md

```diff
@@ -9,13 +9,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-- expirationd configuration via Tarantool Cartridge role configuration (#131).
-
 ### Changed
 
 ### Fixed
 
-- Incorrect check of the Tarantool version in the tests to determine a bug in
+## 1.3.1 - 2023-01-17
```

Comment:
Why not minor? Is it incompatible?

Reply:
Sorry, I meant minor

Reply:
I look at it as: we added a missed part of a feature. It might have sense without a new release, since the previous one:

```
1.3.0:
The release adds the ability to configure the expirationd using Tarantool Cartridge role configuration.
```

Reply:
Well, ok, it makes sence

Reply:
> But it contradicts with your presentation of this change in the changelog: it is in the 'new features' section.

Thank you. I've updated the changelog to make it less confusing.